testsuite/forge_wrapper_core/cluster.py

```diff
@@ -157,6 +202,8 @@ class ForgeJob:
     name: str
     phase: str
     cluster: ForgeCluster
+    num_validators: int = 0
```

Comment:
Having this set to 0 is not quite true its probably better as an optional (typign.Optional) and set initially to None

But pedantics aside this works